src/test/java/com/google/javascript/gents/singleTests/comments.js

```diff
@@ -40,7 +40,7 @@ var foo = function(deleted, notdeleted) { return deleted + notdeleted; };
  * @package {number}
  * @const {number}
  */
-var x;
+var x = 1;
```

Comment:
huh... this is special.  

i'm not sure this can be both a `@constructor` and `@const {number}`